### DISCUSSION TITLE
when closing a parent tab, make the group tab keep its title

### DISCUSSION THREAD

**Discussion by aeshna-cyanea** [piroor/treestyletab#3681] [Ideas]
If you configure the add-on so that when a the parent tab of a group is closed it turns into a group tab, the new group tab takes the title of its first child. 

I would like an option for the new group tab to take the title (and contain a link to the url) of the tab it is replacing instead.

before
![image](https://github.com/user-attachments/assets/b086fd97-663b-4480-86a1-f0ceb7022786)
after
![image](https://github.com/user-attachments/assets/e67bbcb5-e2ec-4d1d-80ed-e2e57ae9440b)

so in the second picture, i would like the tab to still be titled "addon manager" rather than "new discussion"

**Comment by piroor** [comment]
There is no such option, so currently you need to rename the tab by your hand. Switch to the group tab and click the title in the content area, then you'll see a text box to edit the title.

Sadly your idea contains a blocker. On your scenario, the closed "about:addons" tab is replaced with a group tab containing a link to the URI "about:addons". But the link won't work actually - when you click the link Firefox blocks loading of it due to security.
